mleap-xgboost-runtime/src/test/scala/ml/combust/mleap/xgboost/runtime/testing/BundleSerializationUtils.scala

```diff
@@ -0,0 +1,35 @@
+package ml.combust.mleap.xgboost.runtime.testing
+
+import java.io.File
+
+import ml.combust.bundle.BundleFile
+import ml.combust.bundle.serializer.SerializationFormat
+import ml.combust.mleap.runtime.{MleapContext, frame}
+import ml.combust.mleap.runtime.frame.Transformer
+import resource.managed
+
+trait BundleSerializationUtils {
+
+  def serializeModelToMleapBundle(transformer: Transformer): File = {
+    import ml.combust.mleap.runtime.MleapSupport._
+
+    new File("/tmp/mleap/spark-parity").mkdirs()
```

Comment:
perhaps "/tmp/mleap/xgboost-runtime-parity" as this doesn't test anything spark related?

Reply:
Oh yes, leftover from copy-pasting from the spark tests. Let me update it